src/java/org/apache/cassandra/tools/nodetool/InvalidatePermissionsCache.java

```diff
@@ -38,7 +38,7 @@
 @Command(name = "invalidatepermissionscache", description = "Invalidate the permissions cache")
 public class InvalidatePermissionsCache extends NodeToolCmd
 {
-    @Arguments(usage = "[<user>]", description = "A specific user for whom permissions need to be invalidated")
```

Comment:
I renamed user to role here to be coherent with the VT. Also "user" term seems to be outdated since everything relies onto roles now.